spec/spec_helper.rb

```diff
@@ -0,0 +1,106 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
Some people like to leave this in, and others like to take it out. So I just left the comments in, I find it's easier than looking up the docs.